test/operators.js

```diff
@@ -296,31 +282,31 @@ test('live votesFor', (t) => {
     db.publish(voteMsg1, (err, v1) => {
       t.error(err, 'no err')
 
-      db.onDrain(() => {
-        let i = 0
-        pull(
-          db.query(
-            and(votesFor(postMsg.key)),
-            live(),
-            toPullStream(),
-            pull.drain(
-              (result) => {
-                if (i++ == 0) {
-                  t.equal(result.key, v1.key)
-
+      let i = 0
+      pull(
+        db.query(
+          and(votesFor(postMsg.key)),
+          live(),
+          toPullStream(),
+          pull.drain(
+            (result) => {
+              if (i++ == 0) {
+                t.equal(result.key, v1.key)
+
+                setTimeout(() => {
```

Comment:
why the settimeout here?

Reply:
Oh, I think that's an artifact of debugging left around. I'll try to remove it